doc/user/content/get-started.md

```diff
@@ -73,237 +62,465 @@ Download and install the [PostgreSQL installer](https://www.postgresql.org/downl
 
-1. To see the results:
+1. Query the view again:
 
     ```sql
-    SELECT * FROM item_summary ORDER BY revenue DESC LIMIT 5;
+    SELECT * FROM avg_bids;
     ```
+    <p></p>
 
-### Work with time windows
+    ```nofmt
+            item        |    average_bid
+    --------------------+--------------------
+     Custom Art         | 49.783986655546286
+     Gift Basket        |  49.93436483689761
+     City Bar Crawl     |  49.93733653007847
+     Best Pizza in Town |  50.43617136074242
+     Signed Memorabilia |  50.09202958093673
+    (5 rows)
+
+    Time: 26.403 ms
+    ```
 
-In streaming, time keeps ticking along — unlike in batch, where data is frozen at query time. [Temporal filters](/sql/patterns/temporal-filters/) allow you to define time-windows over otherwise unbounded streams of data.
+    You'll see the average bids continue to change, but now that the view is indexed and results are pre-computed and stored in memory, latency is down to 26 milliseconds!
 
-1. Create a view `item_summary_5min` that keeps track of any items that had orders in the past 5 minutes:
+1. One thing to note about indexes is that they exist only in the cluster where they are created. To experience this, switch to the `default` cluster and query the view again:
 
     ```sql
-    CREATE VIEW item_summary_5min AS
-    SELECT *
-    FROM item_summary
-    WHERE mz_now() >= latest_order
-    AND mz_now() < latest_order + INTERVAL '5' MINUTE;
+    SET CLUSTER = default;
+    ```
+
+     ```sql
+    SELECT * FROM avg_bids;
     ```
+    <p></p>
+
+    ```nofmt
+            item        |    average_bid
+    --------------------+--------------------
+     Custom Art         |  49.76620397600282
+     Gift Basket        | 49.850028105677346
+     City Bar Crawl     |  50.08233974737339
+     Best Pizza in Town |  50.46824567514223
+     Signed Memorabilia |  50.12977674688315
+    (5 rows)
+
+    Time: 846.322 ms
+    ```
+
+    Latency is high again because the index you created on the view exists only inside the `compute_qck` cluster. In the `default` cluster, where you are currently, you don't have access to the index's pre-computed results. Instead, the view is again retrieving data from durable storage and computing the results at query-time.
 
-    “In the past 5 minutes” is a moving target that changes as time ticks along. The `mz_now()` function is used to keep track of the logical time for your query (similarly to `now()` in other databases, as explained more in-depth in ["now and mz_now functions"](/sql/functions/now_and_mz_now/)). As time advances, records progressively falling outside this 5-minute window will be expired, and new records that satisfy the time constraint will be used in the view.
+1. There are lots of cases where you'll want results to be accessible from multiple clusters, however. In such cases, you use [materialized views](/overview/key-concepts/#materialized-views).
```

Comment:
```suggestion

1. In many cases, you need multiple clusters to have access to results across your environment. To achieve this, you use [materialized views](/overview/key-concepts/#materialized-views).
```

Reply:
Done, partially. Given the paragraph prior, I feel like I need to leave some sense of transition, even though this isn't idea in this numbered format.